### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by vincentye9682** [getlantern/lantern-forum-cn#1026] [General]
怎么现在蓝灯这么不稳定了？时断时续的，没法用！申请退款

mail:  yizunquan@163.com

Payment time:  2024-11-07 11:20:28

Payment method:  CMB Credit Card(7419)

游戏充值订单号: P01202411071119396916555

Full name of payee: 广州饶铭网络科技有限公司

Order No.: 2024110722001476771458238537

Merchant order No.: 2024110711194145426

**Comment by MaxMKMax** [comment]
@vincentye9682 您好. 已经帮您提交好了. 请您耐心等待. 谢谢.

**Reply by vincentye9682** [reply]
> @vincentye9682 您好. 已经帮您提交好了. 请您耐心等待. 谢谢.
你好，怎么还没任何进展？

**Reply by vincentye9682** [reply]
@Mike777Khoo 靓仔，什么时候能退款？超过一个月就不能退，等下都被拖过一个月了，还怎么退？

**Reply by vincentye9682** [reply]
> @vincentye9682 您好. 已经帮您提交好了. 请您耐心等待. 谢谢.

@Mike777Khoo 马上就2周了，还没任何进展，怎么回事啊，兄弟？

**Reply by vincentye9682** [reply]
@Mike777Khoo 什么情况现在，兄弟？
